PyInstaller/lib/modulegraph/util.py

```diff
@@ -115,3 +116,33 @@ def guess_encoding(fp):
             return m.group(1).decode('ascii')
 
     return default_encoding
+
+
+# Type of all code objects.
+# No need to re-evaluate the type
+code_object_type = type(compile("a = True", "string", "exec"))
+
+
+if PY2:
+    def enumerate_instructions(module_code_object):
+        for instruction in get_instructions(module_code_object):
+            yield instruction
```

Comment:
I suggest yielding a `Instruction(opcode.NOP, None)` after each code-block. This will ensure a discontinuity when scanning the code. This may not be a problem now, but if the byte-code or our processing changes, the NOP might be important.

I assume this `Instruction(opcode.NOP, None)` can be a constant / singleton.